cmd/commands/util.go

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
I am sort of torn about whatever this is the correct place to put this.

I am sort of leaning towards placing this somewhere where the `clio` but I am not sure. Im putting it out there, maybe you'll think of something.